linkerd/core/src/main/scala/io/buoyant/linkerd/Router.scala

```diff
@@ -101,26 +107,39 @@ trait RouterConfig {
   var baseDtab: Option[Dtab] = None
   var failFast: Option[Boolean] = None
   var timeoutMs: Option[Int] = None
+  var dstPrefix: Option[String] = None
```

Comment:
Good question.

There's no reason that 2 routers couldn't share a dtab (so that they can be controlled centrally), but have distinct entry points...

Really, I think that we should drop `dstPrefix` in favor of `identifier` modules (#120).  Individual Identifiers could chose to expose a `prefix` configuration, or not.